src/components/atoms/CardBody/CardBody.ts

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+
+export const CardBody = styled.a`
+	color: black;
```

Comment:
@devan-huapaya @AFrey95 @cosmovane I think we should some sort of theme system in place.

This is not tested but you will get the idea, let me know what do you think:

Constants:

`export const COLORS = {
    primary: '#5987EB',
    secondary: '#BE78F0'
}

export const FONTS = {
    families: {
        primary: "'Roboto', sans-serif"
    },
    colors: {
        default: 'black'
    },
    sizes: {
        sm: 0.875; /* 14px */
        md: 1; /* 16px */
        lg: 0.875; /* 18px */
    }
}

export const TRANSITIONS = {
    default: '.3s';
}`

Global style:

`import { COLORS, FONTS, TRANSITIONS } from "../constants"
import styled, { css } from "styled-components"

function createCSSVariables(prefix, variables, suffix) {
    let output = ''

    for (const key in variables) {
        output += `--${prefix}-${key}: ${variables[key]}${suffix}`
    }

    return css`${output}`
}

const GlobalStyles = createGlobalStyle`
  html {
    ${createCSSVariables('color', COLORS)}
    ${createCSSVariables('font-family', FONTS.families)}
    ${createCSSVariables('font-color', FONTS.colors)}
    ${createCSSVariables('font-size', FONTS.sizes)}
    ${createCSSVariables('transition', TRANSITIONS)}
  }
`;`

Styled component:

`import styled from 'styled-components'

export const CardBody = styled.a`
    color: var(--font-color-default);
    transition: var(--transition-default);
    
    &:hover,
    &:focus,
    &:active {
        color: var(--color-primary);
        border-color: var(--color-secondary);
    }
``